### PR TITLE
checkout-wrapper: fix creation on open modal

### PR DESCRIPTION
**Pull request by vcapretz**
when opening the modal the user can give us the target, or we can create
a default one

now in both cases we check if it exists, and if so, we don't create a
new one

😉

## Description

Closes https://github.com/pagarme/mercurio/issues/108
 

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ import ErrorPage from './pages/Error'
 import createElement from './utils/helpers/createElement'
 import setTheme from './utils/helpers/setTheme'
 import setColors from './utils/helpers/setColors'
+import DEFAULT_COLORS from './utils/data/colors'
```
Comment (thiagommedeiros) [inline src/index.js]: Pq usou uppercase e underline?

Comment (vcapretz) [reply src/index.js]: pensei em usar esse padrão pra definir constantes, fica mais legível no meio do código sabe? o que acha?

Comment (augusto-jm-amaral) [reply src/index.js]: Acho que fica legal uppercase :)

Comment (allangrds) [reply src/index.js]: Também gosto

**Pull request merged**